shorebird_code_push/README.md

```diff
@@ -4,6 +4,38 @@
 
 A Dart library that allows Flutter apps to get information about code push updates.
 
+## Usage
+
+```dart
+import 'package:shorebird_code_push/shorebird_code_push.dart';
+
+final shorebirdCodePush = ShorebirdCodePush();
+
+// Gets the current patch version, or null if no patch is installed.
+final currentPatchversion = shorebirdCodePush.currentPatchVersion();
+
+// Checks whether a patch is available to install.
+final isUpdateAvailable = await shorebirdCodePush.checkForUpdate();
+```
+
+## Developing
+
+### FFI
+
+The Dart code in this library communicates with the Updater (part of Shorebird's
+Flutter engine) via FFI.
+
+For an Updater function to be visible to the Dart code, it must:
+
+1. Be declared in c_api.rs as `pub extern "C"`.
+    1. This will add the function to the `library/include/updater.h` header
+    file, which is generated by [cbindgen](https://github.com/mozilla/cbindgen)
+    when the Updater is built.
+1. Be listed in
```

Comment:
This is only for Android (iOS will have a different file I suspect).

Reply:
Addressed